aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -219,6 +219,7 @@ module aptos_framework::coin {
         type_info::account_address(&type_info)
     }
 
+    #[view]
```

Comment:
Will there be other functions in aptos framework that we want to mark as view function? cc @movekevin

Reply:
Probably but this would be out-of-scope for this PR.

Reply:
+1. We can comb through functions soon and add #[view] in a separate PR